source/shared/net/concepts/concepts-transactions.rst

```diff
@@ -0,0 +1,45 @@
+.. _transactions-overview:
+
+=====================
+Transactions overview
+=====================
+
+.. contents::
+    :local:
+    :backlinks: none
+
+A transaction on the Concordium blockchain is an operation which applies some change to the chain. All transactions are recorded on the chain and once recorded, they are immutable. A transaction always has one sender :ref:`account<glossary-account>` and is signed using the keys of this account.
+
+The most basic transaction is the GTU transfer that is used to send GTU from one account to another. However, there are several transaction types on the Concordium blockchain:
+
+- Send GTU
+- Send GTU with a schedule
+- Shield GTU
+- Unshield GTU
+- Make shielded transfer
+- Add baker
+- Remove baker
+- Update baker stake
+- Update restake earnings
+- Update baker keys
+- Update account credentials
+
+You can make transactions using either the Desktop Wallet, the Mobile Wallet, or the Concordium Client. Note that the Mobile Wallet doesn’t support all transaction types, and that you need a Ledger device to submit transactions from the Desktop Wallet.
+
+- Mobile Wallet: supports send GTU, shield GTU, and unshield GTU.
+- Desktop Wallet: supports all transaction types
```

Comment:
This is not currently true, it does not support smart contract ones as far as I'm aware.